alerter.py

```diff
@@ -16,10 +18,11 @@ def alert_in_celcius(farenheit):
         # However, this code doesn't count failures!
         # Add a test below to catch this bug. Alter the stub above, if needed.
         global alert_failure_count
-        alert_failure_count += 0
+        alert_failure_count += 1
 
 
 alert_in_celcius(400.5)
 alert_in_celcius(303.6)
+assert(alert_failure_count == 0)

```

Comment:
Test case is expecting zero alert failure count. Is this the expected outcome of production code? In that case there should be tests that expect non zero failure count too.